llvm/test/CodeGen/AIE/aie2/ra/waw_reg_renaming_loop.mir

```diff
@@ -0,0 +1,579 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 4
+# NOTE: Example file for Write After Write Register Renaming in Loop test
+#
+# This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# (c) Copyright 2024 Advanced Micro Devices, Inc. or its affiliates
+
+# RUN: llc -mtriple=aie2 -verify-machineinstrs --run-pass=greedy,aie-waw-reg-rewrite,virtregrewriter %s -o - | FileCheck %s
+# RUN: llc -mtriple=aie2 -verify-machineinstrs --run-pass=greedy,virtregrewriter %s -o - | FileCheck %s --check-prefix=DISABLE-PASS
+#
+---
```

Comment:
Maybe a small description of each test can be included, forecasting the expected result.

Reply:
added a comment for each test, to help clarify what each test does